modules/configuration/test/ConfigurationTest.php

```diff
@@ -37,9 +37,71 @@ public function testConfigurationPageLoads()
     {
         $this->webDriver->get($this->url . "?test_name=configuration");
 
-        $bodyText = $this->webDriver
-            ->findElement(WebDriverBy::cssSelector("body"))->getText();
-        $this->assertContains("Configuration", $bodyText);
+        $breadcrumbText = $this->webDriver->findElement(
+            WebDriverBy::xPath("//div[@class='page-content inset']/div/a/label")
+        )->getText();
+        $this->assertEquals("Configuration", $breadcrumbText);
     }
+
+    /**
+     * Verify that Config module appears in Admin main menu only
+     * if the user has permission "config".
+     *
+     * @return void
+     */
+    public function testConfigurationMenuDisplayWithPermission()
+    {
+        $this->setupPermissions(array('config'));
+
+        $this->webDriver->navigate()->refresh();
+
+        $configMenu = $this->webDriver->findElements(
+            WebDriverBy::xPath(
+                "
+                //ul[@class='nav navbar-nav']
+                //a[contains(text(), 'Admin')]
+                /..
+                /ul[@class='dropdown-menu']
+                //a[contains(text(), 'Configuration')]
```

Comment:
Can this be turned into a helper function in the LorisIntegrationTest base class that other people can use? Something like assertMenuItemPresent($category, $menu) (and assertMenuItemNotPresent($category, $menu)) that people can use for other modules..